### DISCUSSION TITLE
side icon with favourites directories.

### DISCUSSION THREAD

**Discussion by Luis-Lourenco** [filebrowser/filebrowser#2067] [Ideas]
Hello,

its possible on config settings.json or on webgui select some directory as favourites, to appear together above myfiles?

**Comment by ramiresviana** [comment]
There's no feature for favorites directories on File Browser.

**Comment by bazmattaz** [comment]
Me too i would love this feature. Just a way to save folders would be great

**Comment by ecker00** [comment]
Just chiming in that this is requested quite a lot, hope to see it implemented.

To gather all the threads on this topic. 
#2263 
#2429
#2141
#2076
